build_rules.gradle

```diff
@@ -593,6 +593,108 @@ ext.applyJavaNature = {
   }
 }
 
+///
+class JavaPerformanceTestConfiguration {
+  String homeDir = System.getProperty('user.home')
+  String pkbLocation = System.getProperty('pkbLocation')
+
+  String logLevel = System.getProperty('logLevel', 'INFO')
+  String gradleBinary = System.getProperty('gradleBinary', './gradlew')
+  String isOfficial = System.getProperty('official', 'true')
+  String benchmarks = System.getProperty('benchmarks', 'beam_integration_benchmark')
+
+  String beamPrebuilt = System.getProperty('beamPrebuilt', 'true')
+  String beamSdk = System.getProperty('beamSdk', 'java')
+
+  String timeout = System.getProperty('itTimeout', '1200')
+
+  String kubeconfig = System.getProperty('kubeconfig', System.getProperty('user.home') + '/.kube/config')
+  String kubectl = System.getProperty('kubectl', 'kubectl')
+  String kubernetesScripts = System.getProperty('kubernetesScripts')
+
+  String integrationTestPipelineOptions = System.getProperty('integrationTestPipelineOptions')
+  String optionsConfigFile = System.getProperty('beamITOptions')
+
+  String integrationTest = System.getProperty('integrationTest')
+  String itModule = System.getProperty('itModule')
+  String runner = System.getProperty('integrationTestRunner')
+
+  String extraProperties = System.getProperty('beamExtraProperties')
+}
+
+ext.createJavaPerformanceTest = {
+  println "createJavaPerformanceTest with " + (it ? "$it" : "default configuration") + " for project $project.name"
+  // Use the implicit it parameter of the closure to handle zero argument or one argument map calls.
+  JavaPerformanceTestConfiguration configuration = it ? it as JavaPerformanceTestConfiguration : new JavaPerformanceTestConfiguration()
+
+  for runner in "dataflow", "direct":
+  configuration {
+    javaPerfTest
+  }
```

Comment:
I have trouble understanding what is this loop for. The overall config works as it should without it.